src/test/java/bio/terra/service/filedata/DrsTest.java

```diff
@@ -170,6 +180,29 @@ public void drsHackyTest() throws Exception {
     validateBQJobUserRolePresent(
         Arrays.asList(snapshotIamRoles.get(IamRole.STEWARD), snapshotIamRoles.get(IamRole.READER)));
 
+    Optional<DRSAccessMethod> drsAccessMethod =
+        drsObjectFile.getAccessMethods().stream()
+            .filter(accessMethod -> accessMethod.getType().equals(TypeEnum.GS))
+            .findFirst();
+
+    if (drsAccessMethod.isEmpty()) {
+      Assert.fail("missing access method for DRS object");
+    }
+
+    String drsAccessId = drsAccessMethod.get().getAccessId();
+    bio.terra.model.DRSAccessURL drsAccessURL =
+        drsService.getAccessUrlForObjectId(authenticatedCustodianRequest, drsObjectId, drsAccessId);
+
+    try (CloseableHttpClient client = HttpClients.createDefault()) {
+      HttpUriRequest request = new HttpHead(drsAccessURL.getUrl());
+      request.setHeader(
```

Comment:
I think the point of the signed URL is that you don't need the authorization.  You should take this out of the request header